backend/hitas/models/apartment.py

```diff
@@ -30,12 +30,10 @@
 
 class ApartmentState(Enum):
     FREE = "free"
-    RESERVED = "reserved"
     SOLD = "sold"
 
     class Labels:
         FREE = _("Free")
-        RESERVED = _("Reserved")
         SOLD = _("Sold")
```

Comment:
Since we are changing this from a '3 value enum' to a enum with only 2 values, it doesn't make sense to keep it as an enum anymore. I would do a more major and change `state` to a `is_sold` (boolean), as even the `state` name does not fit that well anymore

Reply:
I initially had this a boolean but decided to try to minimize required changes in the frontend, and so I kept this as an enum. We can go with the boolean route though.

Reply:
If you want, I can do the frontend part to this same PR when the backend is using booleans, just ping me :)

Reply:
Backend changes to `is_sold` now in this branch, go ahead with the frontend!